NOTICE.md

```diff
@@ -7,5 +7,5 @@ http://www.eclipse.org/legal/epl-2.0.
 SPDX-License-Identifier: EPL-2.0
 
 ## Contributors:
-*   Robert Bosch GmbH - initial API and implementation
-*   Microsoft Corporation - initial API and implementation
```

Comment:
why is EPL-2 mentioned here? shouldn't this be apache2?

did microsoft really contribute something to this repo?

Reply:
As for EPL-2, Good question - as you can see - it was there before me :)
It is an error - I will change it on Apache-2.

As for Mircosoft, some files were taken from repositories, where MS was previously contributed. 
To avoid any issues - I would like to leave them.

Reply:
I'm also not sure that Microsoft contributed to this repo. But I guess some content inside here will originate from the Microsoft cooperation, so lets keep them in here.